Cargo.toml

```diff
@@ -4,6 +4,10 @@ version = "0.0.1"
 authors = ["Antoine Poinsot <darosior@protonmail.com>"]
 edition = "2018"
 
+[lib]
+name = "common"
+path = "src/common/lib.rs"
```

Comment:
why not a different workspace member, importing only public modules from revaultd ?
And I am not a fan of this kind of path:
src/bin/revaultd/bitcoind/actions.rs

Reply:
Pushed a new commit for a new layout which gets rid of `src/bin` in favour of `src/daemon` and `src/cli`